java/Nikiforov-Oleksandr/src/com/zoolatech/lecture2/tasks/_1/Calculcator.java

```diff
@@ -0,0 +1,42 @@
+package com.zoolatech.lecture2.tasks._1;
+
+public class Calculcator {
+    private float value;
+
+    public Calculcator(float value){
+        this.value = value;
+    }
+
+    public float getValue() {
+        return value;
+    }
+
+    public void setValue(float value) {
+        this.value = value;
+    }
+
+    public float multiplication(float variable){
```

Comment:
`variable` might be not the best naming for the method argument, try to think about a better one